### DISCUSSION TITLE
l7CityBuildingLayer加载建筑窗户不生效

### DISCUSSION THREAD

**Discussion by ikun159357** [antvis/L7#2277] [Q&A]
配置代码如下：
 wuhan_buildingLayer.source(buildData.data).size('Elevation', h => h).color("rgb(242,242,250,1.0)")
        .animate({
            enable: true,
        })
        .active({
            color: '#0ff',
            mix: 0.5
        })
        .style({
            opacity: 0.7,
            baseColor: 'rgb(16, 16, 16)',
            windowColor: 'rgb(30, 60, 89)',
            brightColor: 'rgb(255, 176, 38)',
            sweep: {
                enable: true,
                sweepRadius: 2,
                sweepColor: '#1990FF',
                sweepSpeed: 0.5,
                sweepCenter: [114.30, 30.50]
            }
        }).filter('Elevation', h => h > 35)//筛选楼房高于35米
    scene.addLayer(wuhan_buildingLayer)
![Snipaste_2024-01-30_10-06-16](https://github.com/antvis/L7/assets/140690304/97a09a7a-a1fc-48d8-9bab-dd3eb9a4c4bd)
![Snipaste_2024-01-30_10-06-42](https://github.com/antvis/L7/assets/140690304/c00c0837-fca0-4ce9-9b50-fa17683ea57d)

**Comment by lzxue** [comment]
官网Demo 原样换一下数据呢？

**Reply by ikun159357** [reply]
感谢，我重新试了一下，貌似color里面rgb双引号改成单引号就可以了
